ddtrace/internal/utils/http.py

```diff
@@ -19,7 +19,7 @@
 from ddtrace.internal.utils.cache import cached
 
 
-_W3C_TRACESTATE_INVALID_CHARS_REGEX = r",|;|:|[^\x20-\x7E]+"
+_W3C_TRACESTATE_INVALID_CHARS_REGEX = r",|;|[^\x20-\x7E]+"
```

Comment:
Is this related to the `~/_/=/~` replacement changes? If it is, can you add it to the PR description? Thanks!

Reply:
Yup just added, thanks!